src/ensembl/tests/fixtures/entity-viewer/transcript.ts

```diff
@@ -92,22 +94,26 @@ export const createTranscriptMetadata = (fragment: Partial<TranscriptMetadata> =
   };
 };
 
+export const createExternalReference = (
```

Comment:
Probably better move it to its own file; external references are not exclusive to transcripts.

Reply:
move it to shared ?